src/encoder.rs

```diff
@@ -1970,9 +1970,10 @@ fn encode_tile(sequence: &mut Sequence, fi: &FrameInvariants, fs: &mut FrameStat
 
             // Encode SuperBlock
             if fi.config.speed == 0 {
+                let pmv = MotionVector { row: 0, col: 0 };
```

Comment:
Why don't we do this for both directions?

Reply:
Can you explain what do you mean by "both directions"? 
If so, two reference frames? It is not landed yet.

Reply:
Er nevermind, I didn't understand the patch.

Reply:
It would be nice to predict from our superblock neighbors, e.g., median-of-three, but even just the left (or top) superblock would do. But no objections to landing this patch as-is and trying that in a separate patch.

Reply:
Thanks! Your suggestion of having SB neighbor's MVs as reference for ME search center is w/o doubts we want to have it. I (or someone who cannot wait my rust code writing speed) will write it as a separate patch.